pac/atsamd11c/src/lib.rs

```diff
@@ -914,18 +913,22 @@ pub struct Peripherals {
     pub WDT: WDT,
 }
 impl Peripherals {
-    #[doc = r"Returns all the peripherals *once*"]
+    #[doc = r" Returns all the peripherals *once*."]
+    #[cfg(feature = "critical-section")]
```

Comment:
The optional dependency is forced by this new addition to the generated pac. One can still use steal if one wants to avoid the dependency's, but has to use unsafe code.

Reply:
Ah fair enough. Though I don't really see a case where someone wouldn't want to use the critical section feature - I would suggest enabling it in the HAL for all PACs. That way users and libraries can use the`critical-section` lib without having to worry about cargo features. After all, it's not like there should be another critical section provider in the dependency tree anyways - what do you think?